src/main/java/com/rocketden/main/model/problem/Problem.java

```diff
@@ -0,0 +1,45 @@
+package com.rocketden.main.model.problem;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity // This tells Hibernate to make a table out of this class
+@Getter
+@Setter
+public class Problem {
```

Comment:
We should add a difficulty level, and problem type, to these problems. This could be done now or in the future, but it will be necessary if people want to populate the game with particular problem sets.

Reply:
Yep, adding the difficulty in #71 - it doesn't have the problem type yet (I think that could be post-MVP since we prob won't have enough problems to support that yet), but I'm open to adding that too if we think it'll be useful now

Reply:
That makes sense, I'm open to adding that during the MVP but no need to add it in this PR, so all good either way.